zapier/triggers/list_forms.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  operation: {
+    perform: {
+      headers: { Accept: 'application/json' },
+      params: {
+        api_key: '{{bundle.authData.api_key}}',
+        workspace_id: '{{bundle.inputData.workspace_id}}',
+      },
+      removeMissingValuesFrom: { body: false, params: false },
+      url: '{{process.env.BASE_URL}}/api/zapier/forms',
+    },
+    inputFields: [
+      {
+        key: 'workspace_id',
+        type: 'string',
+        dynamic: 'list_workspaces.id.name',
+        label: 'Workspace',
+        required: true,
+        list: false,
+        altersDynamicFields: false,
+      },
+    ],
+    sample: { id: 1, name: 'My Form' },
+    outputFields: [
```

Comment:
Can we only replace the ID with the form slug?
![You can search only within loaded results  Load more to search for](https://github.com/user-attachments/assets/593f37e7-a149-42f1-9bc9-1867dace5351)